hyperspy/_signals/complex_signal.py

```diff
@@ -123,7 +114,8 @@ def change_dtype(self, dtype):
             raise AttributeError(
                 'Complex data can only be converted into other complex dtypes!')
 
-    def angle(self, deg=False):
+    @format_title('angle')
+    def angle(self, angle, deg=False):
```

Comment:
I'll have to check, but it's possible that it does not break this - this particular code is only called internally, and the public interface is [here](https://github.com/to266/hyperspy/blob/ENH_lazy_signal/hyperspy/_signals/complex_signal.py#L256) and [here](https://github.com/to266/hyperspy/blob/ENH_lazy_signal/hyperspy/_signals/complex_signal.py#L298)